eng/.docsettings.yml

```diff
@@ -21,42 +21,97 @@ required_readme_sections:
   - ^Next steps$
   - ^Contributing$
 known_presence_issues:
-  - ['sdk/keyvault','#5499']
-  - ['sdk/servicebus','#5499']
-  - ['sdk/eventhub','#5499']
-  - ['sdk/eventgrid/Microsoft.Azure.EventGrid','#5499']
-  - ['sdk/operationalinsights/Microsoft.Azure.OperationalInsights','#5499']
-  - ['sdk/hdinsight/Microsoft.Azure.HDInsight.Job','#5499']
-  - ['sdk/alertsmanagement/Microsoft.Azure.Management.AlertsManagement','#5499']
-  - ['sdk/applicationinsights/Microsoft.Azure.ApplicationInsights.Query','#5499']
-  - ['sdk/containerregistry/Microsoft.Azure.ContainerRegistry','#5499']
-  - ['sdk/cognitiveservices/Language.LUIS.Runtime','#5499']
-  - ['sdk/cognitiveservices/Search.BingEntitySearch','#5499']
-  - ['sdk/cognitiveservices/Search.BingWebSearch','#5499']
-  - ['sdk/cognitiveservices/Knowledge.QnAMaker','#5499']
-  - ['sdk/cognitiveservices/AnomalyDetector','#5499']
-  - ['sdk/cognitiveservices/Search.BingVisualSearch','#5499']
-  - ['sdk/cognitiveservices/Search.BingCustomSearch','#5499']
-  - ['sdk/cognitiveservices/Search.BingImageSearch','#5499']
-  - ['sdk/cognitiveservices/Vision.FormRecognizer','#5499']
-  - ['sdk/cognitiveservices/Search.BingVideoSearch','#5499']
-  - ['sdk/cognitiveservices/Search.BingNewsSearch','#5499']
-  - ['sdk/cognitiveservices/Personalizer','#5499']
-  - ['sdk/cognitiveservices/Vision.ComputerVision','#5499']
-  - ['sdk/cognitiveservices/Vision.CustomVision.Training','#5499']
-  - ['sdk/cognitiveservices/Vision.Face','#5499']
-  - ['sdk/cognitiveservices/Search.BingAutoSuggest','#5499']
-  - ['sdk/cognitiveservices/Language.LUIS.Authoring','#5499']
-  - ['sdk/cognitiveservices/Vision.ContentModerator','#5499']
-  - ['sdk/cognitiveservices/Vision.CustomVision.Prediction','#5499']
-  - ['sdk/cognitiveservices/Language.SpellCheck','#5499']
-  - ['sdk/cognitiveservices/Language.TextAnalytics','#5499']
-  - ['sdk/cognitiveservices/Search.BingLocalSearch','#5499']
-  - ['sdk/cognitiveservices/Search.BingCustomImageSearch','#5499']
-  - ['sdk/cognitiveservices/FormRecognizer','#5499']
-  - ['sdk/batch/Microsoft.Azure.Batch.FileStaging','#5499']
-  - ['sdk/graphrbac/Microsoft.Azure.Graph.RBAC','#5499']
-  - ['sdk/search','#5499']
+  - ['sdk/eventgrid/Microsoft.Azure.EventGrid/README.md','#5499']
```

Comment:
This list would be much easier to scan if you keep it alphabetized.

Reply:
Sorted. Thanks for pointing that out.